sentence_transformers/SentenceTransformer.py

```diff
@@ -450,8 +450,9 @@ def _create_model_card(self, path: str, model_name: Optional[str] = None, train_
             fOut.write(model_card.strip())
 
     def save_to_hub(self,
-                    repo_name: str,
+                    repo_id: str,
```

Comment:
I think these param rename/additions are backwards breaking right?

Reply:
Only when calling with keyword arguments, not with positional arguments. I'll look at a decorator-based fix tomorrow to ideally keep the IDE-suggested signature using `repo_id`.

Reply:
The addition of token would break positional arguments for private/commit/local_model_path/etc

Reply:
That is only avoided by introducing the argument at the end, which is undesirable, or via some very messy use of `*args` and redistributing the captured args.

Positional usage is not recommended for keyword arguments here (or anywhere, for that matter). 

Do you have a fix in mind?